src/library/librarycontrol.cpp

```diff
@@ -509,6 +514,11 @@ void LibraryControl::slotMoveHorizontal(double v) {
     emitKeyEvent(QKeyEvent{QEvent::KeyPress, key, Qt::NoModifier, QString(), false, times});
 }
 
+void LibraryControl::slotCurrentColumn(double v) {
+qDebug() << "UUUUUUUUUUUUUUUUUUUUUUUUUUUUU" << "LibraryControl::slotCurrentColumn" << "UUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUUU";
```

Comment:
I suggest to use `git add -p` to avoid committing debugging leftovers in the future.